server/services/visitNotificationsService.ts

```diff
@@ -0,0 +1,47 @@
+import { VisitsReviewListItem } from '../@types/bapv'
+import { HmppsAuthClient, OrchestrationApiClient, RestClientBuilder } from '../data'
+import { NotificationCount } from '../data/orchestrationApiTypes'
+import { prisonerDateTimePretty } from '../utils/utils'
+
+export default class VisitNotificationsService {
```

Comment:
Created new service to deal with visit notifications; should be a better fit as the journey is built out.